### ISSUE TITLE
'cfg.OPT.DECAY_EPOCHS' missing

### ISSUE THREAD

**Issue by bbb801** [VinBDI-MedicalImagingTeam/midl2020-cnnlstm-ich#3]
Dear Sir,

It seems some parameters are missing. For example, 'cfg.OPT.DECAY_EPOCHS' from 'lr_scheduler.py' can not be found in 'defaults.py'. Also, some parameters' names seem to be changed. Any advice? Thank you.

![image](https://user-images.githubusercontent.com/52306405/95614999-12b49d00-0a9a-11eb-9fad-c864331f9389.png)
![image](https://user-images.githubusercontent.com/52306405/95615064-29f38a80-0a9a-11eb-861c-c8bf0f15e320.png)
![image](https://user-images.githubusercontent.com/52306405/95614933-f3b60b00-0a99-11eb-8c1f-f5a196652eb9.png)

**Comment by dattran2346** [comment]
Hi @bbb801, 

The `decay_warmup` is a legacy scheduler, the setting in the paper use `cosine_warmup` only. So you can ignore that configuration.

Best.
